traveler-ios-ui/TravelerKitUI/ViewControllers/PurchaseDetailsViewController.swift

```diff
@@ -10,19 +10,19 @@ import UIKit
 import TravelerKit
 
 protocol PurchaseDetailsViewControllerDelgate: class {
-    func purchaseDetailsViewControllerDidChangePreferredContentSize(_ controller: PurchaseDetailsViewController)
```

Comment:
same thing here.

BUT

You are correct in your assumption that this vc is no longer needed. So lets clean it up